code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -808,7 +808,10 @@
 	taste_description = "pure electricity"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-/datum/reagent/consumable/liquidelectricity/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume) //can't be on life because of the way blood works.
+/datum/reagent/consumable/liquidelectricity/enriched
```

Comment:
it'd be nice if you added a chem recipe for ethereal blood enrichment

maybe something involving teslium?

Reply:
maybe but not in this pr